pennylane/beta/qnodes/cv.py

```diff
@@ -220,15 +220,17 @@ def _pd_analytic(self, idx, args, kwargs, **options):
                     B_inv = B_inv @ BB.heisenberg_tr(w, inverse=True)
                 Z = B @ Z @ B_inv  # conjugation
 
-                # transform the descendant observables
+                # transform the descendant observables into their derivatives using Z
                 desc = self._op_descendants(op, "O")
-                obs = [
-                    self._transform_observable(x, w, Z) if x in desc else x
-                    for x in self.circuit.observables
-                ]
-
-                # measure transformed observables
-                pd += self.evaluate_obs(obs, unshifted_args, kwargs)
+                obs = [self._transform_observable(x, w, Z) for x in desc]
+                # Measure the transformed observables.
+                # The other observables do not depend on this parameter instance,
+                # hence their partial derivatives are zero.
+                res = self.evaluate_obs(obs, unshifted_args, kwargs)

```

Comment:
Ah, so the main difference is removing the `if x in desc else x`, and replacing all other observable PD's with 0?